src/components/FeedbackForm/FeedbackForm.tsx

```diff
@@ -26,18 +23,18 @@ const FeedbackForm = ({
   >;
 }) => {
   const [rating, setRating] = useState(0);
-  const [isLoading, setIsLoading] = useState(false);
-  const [formError, setFormError] = useState("");
+  const [wizard, setWizard] = useState<{
+    type: "initial" | "loading" | "error";
```

Comment:
This is not ok. We want the `error` type to always have a message. By making it optional, now you may be in `error` but with no message.


So, it should be like this

```
{
  type: "initial"
} | {
  type: "loading"
} | {
  type: "error",
  message: string;
}
```

This way, the error type IS GUARANTEED to have a messge.